api/s2n.h

```diff
@@ -494,6 +494,18 @@ extern int s2n_config_set_initial_ticket_count(struct s2n_config *config, uint8_
 S2N_API
 extern int s2n_connection_add_new_tickets_to_send(struct s2n_connection *conn, uint8_t num);
 
+/**
+ * Returns the number of session tickets issued by the server.
+ *
+ * In TLS1.3, this number can be up to the limit configured by s2n_config_set_initial_ticket_count
+ * and s2n_connection_add_new_tickets_to_send. In earlier versions, this number will be either 0 or 1.
```

Comment:
I had to read this a couple of times to realize you were talking about TLS versions rather than the s2n library version. Not sure if someone else will run into the same thing but I don't think it would hurt to clarify.

```suggestion
 * and s2n_connection_add_new_tickets_to_send. In earlier versions of TLS, this number will be either 0 or 1.
```